C/include/c4Error.h

```diff
@@ -108,6 +112,17 @@ typedef C4_ENUM(int32_t, C4NetworkErrorCode) {
     kC4NetErrUnknown,                   // Unknown error
     kC4NetErrTLSCertRevoked,            // Peer's cert has been revoked
     kC4NetErrTLSCertNameMismatch,       // Peer's cert's Common Name doesn't match hostname
+    kC4NetErrNetworkReset,              // The network subsystem was reset [ENETRESET, retryable]
+    kC4NetErrConnectionAborted,         // The connection was aborted by the OS [ECONNABORTED, retryable]
+    kC4NetErrConnectionReset,           // The connection was reset by the other side [ECONNRESET, retryable]
```

Comment:
Was ECONNRESET (Linux) not a retriable error before this PR?

Reply:
The retryable error list has not changed logically, these errors are just mirroring the POSIX errors in question.